Remove example papi files from the export (#613)
pascalaubry/papi-web: export-data/papi/README.txt

# Français

> Ce dossier est le dossier par défaut des fichiers Papi utilisés par les tournois dans Sharly-Chess.

Reviewer: ```suggestion
Ce dossier est le dossier par défaut des fichiers Papi utilisés par les tournois dans Sharly Chess.
```

Cela peut être configuré à la fois au niveau de l'évènement et du tournoi.

# English

> This directory is the default location for the Papi files used by the tournaments is Sharly-Chess.

Reviewer: ```suggestion
This directory is the default location for the Papi files of the tournaments in Sharly Chess.
```
